flybywiresim/a32nx
# fix(mcdu): Check Take Off Data

## Summary of Changes

- Improved check logic and various small fixes.

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions

Prepare the aircraft as per usual via DIFRIPPS.
- No CHECK TO DATA Message on initial runway, flaps/ths/flex temp entry
- CHECK TO DATA Message + Confirm Prompt on runway change,
- CHECK TO DATA Message on flaps/ths/flex temp change.



## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link
